requirements.txt

```diff
@@ -1,59 +1,58 @@
-git+https://github.com/alphagov/digitalmarketplace-apiclient.git@21.4.1#egg=digitalmarketplace-apiclient==21.4.1
-git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.4.0-alpha#egg=govuk-frontend-jinja==0.4.0-alpha
-
-## The following requirements were added by pip freeze:
-asn1crypto==1.3.0
-blinker==1.4
+#
+# This file is autogenerated by pip-compile
+# To update, run:
+#
+#    pip-compile requirements.in
+#
+asn1crypto==1.3.0         # via cryptography
+blinker==1.4              # via gds-metrics
 boto3==1.11.14
-botocore==1.14.14
-certifi==2019.11.28
-cffi==1.14.0
-chardet==3.0.4
-Click==7.0
+botocore==1.14.14         # via boto3, s3transfer
+certifi==2019.11.28       # via requests
+cffi==1.14.0              # via cryptography
+chardet==3.0.4            # via requests
+click==7.0                # via flask
 contextlib2==0.6.0.post1
 cryptography==2.3.1
-defusedxml==0.6.0
-docopt==0.6.2
-docutils==0.15.2
-Flask-gzip==0.2
-Flask-Script==2.0.6
+defusedxml==0.6.0         # via odfpy
+git+https://github.com/alphagov/digitalmarketplace-apiclient.git@21.4.1#egg=digitalmarketplace-apiclient==21.4.1
+git+https://github.com/alphagov/digitalmarketplace-content-loader.git@7.1.1#egg=digitalmarketplace-content-loader==7.1.1
+git+https://github.com/alphagov/digitalmarketplace-utils.git@51.3.1#egg=digitalmarketplace-utils==51.3.1
+docopt==0.6.2             # via notifications-python-client
+docutils==0.15.2          # via botocore
+flask-gzip==0.2
+flask-login==0.4.1
+flask-script==2.0.6
+flask-wtf==0.14.3
+flask==1.0.4
 fleep==1.0.1
-future==0.18.2
+future==0.18.2            # via notifications-python-client
 gds-metrics==0.2.0
 govuk-country-register==0.5.0
-idna==2.8
+git+https://github.com/alphagov/govuk-frontend-jinja.git@v0.4.0-alpha#egg=govuk-frontend-jinja==0.4.0-alpha
+idna==2.8                 # via cryptography, requests
 inflection==0.3.1
-Jinja2==2.11.1
-jmespath==0.9.4
+itsdangerous==0.24
+jinja2==2.10.3            # via flask
+jmespath==0.9.4           # via boto3, botocore
 mailchimp3==3.0.6
-Markdown==2.6.11
-MarkupSafe==1.1.1
-monotonic==1.5
+markdown==2.6.11
+markupsafe==1.1.1         # via jinja2
+monotonic==1.5            # via notifications-python-client
 notifications-python-client==5.5.1
 odfpy==1.4.1
-prometheus-client==0.2.0
-pycparser==2.19
-PyJWT==1.7.1
-python-dateutil==2.8.1
+prometheus-client==0.2.0  # via gds-metrics
+pycparser==2.19           # via cffi
+pyjwt==1.7.1              # via notifications-python-client
+python-dateutil==2.8.1    # via botocore
 python-json-logger==0.1.11
 pytz==2019.3
-PyYAML==5.3
-requests==2.22.0
-s3transfer==0.3.3
-six==1.14.0
+pyyaml==5.3
+requests==2.22.0          # via mailchimp3, notifications-python-client
+s3transfer==0.3.3         # via boto3
+six==1.14.0               # via cryptography, python-dateutil
 unicodecsv==0.14.1
-urllib3==1.25.8
-Werkzeug==1.0.0
+urllib3==1.25.8           # via botocore, requests
+werkzeug==0.16.1          # via flask
```

Comment:
Back down to 0.16.1?

Reply:
Yeah, pip-compile is better about preventing/resolving conflicts than our old script. If we want to stay on Werkzeug 1.0.0 I need to push alphagov/digitalmarketplace-utils#556 to all the PRs

Reply:
Hmm OK. I don't think we're using any new 1.0.0 features yet.

Reply:
Not yet, but we might end up deleting some code because Werkzeug 1.x does URL slash merging!